Pipfile

```diff
@@ -16,16 +16,18 @@ coverage = "~=4.5.3"
 unittest-xml-reporting = "~=2.5.1"
 
 [packages]
-django = "~=2.2"
+django = "~=2.1"
```

Comment:
Not a fan of this at all. I'm also not sure if this can break things for us if we port back up to 2.2 later. Also, 2.2 is LTS, so it's a good version to stay on.

Looks like django-wiki depends on < 2.2, but I'm not sure if this is an actual "it breaks on 2.2" or "we didn't update the version specifier yet" - judging from [this commit](https://github.com/django-wiki/django-wiki/commit/018074b62800e2ec91abc6e3cb36f73bc1d4f6c1) they didn't add it to the test matrix yet and maybe could theoretically support it. Maybe we can send in a PR to add 2.2 to the test matrix? I can look into that later today.

Reply:
I think there isn't much chance of getting that merged, but if you're hopeful, be my guest.

Could delay the PR quite a bit if we're waiting on it, though.

Reply:
From Discord: We're not gonna wait to solve this before merging the PR, but it's something that should be prioritised after.

Reply:
huh? which discord is this from?

Reply:
Ah okay, fair

Reply:
This has now been resolved.